.github/workflows/playwright.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        example: ["pets"]
+        project: ["examples/pets"]
```

Comment:
NOTE: Had some troubles on running the test on CI because there the sync halts even with small files. Going to enable them when fixing https://github.com/gardencmp/jazz/issues/389